modules/upgrading-postgresql.adoc

```diff
@@ -0,0 +1,49 @@
+:_content-type: PROCEDURE
+[id="upgrading-postgresql"]
+= Updating {productname} and the {productname} and Clair PostgreSQL databases on {ocp}
+
+When updating {productname} 3.8 -> 3.9, the Operator automatically upgrades the existing PostgreSQL databases for Clair and {productname} from version 10 to version 13. 
```

Comment:
In this doc, we mentioned `3.8 -> 3.9` which is absolutely correct, actually we also support  `3.6 -> 3.9`, `3.7 -> 3.9`, should we mention this somewhere as well?

Reply:
Thanks for confirming. I can add this to the upgrade docs.